src/slack-model.jl

```diff
@@ -446,6 +446,46 @@ function NLPModels.jprod_nln!(
   return jv
 end
 
+function NLPModels.jtprod!(
+  nlp::SlackModels,
+  x::AbstractVector{T},
+  v::AbstractVector,
+  jtv::AbstractVector,
+) where {T}
+  @lencheck nlp.meta.nvar x jtv
+  @lencheck nlp.meta.ncon v
+  n = nlp.model.meta.nvar
+  @views jtprod!(nlp.model, x[1:n], v, jtv[1:n])
+  jtv[n + 1:end] .= zero(T) # Just for init purpose
```

Comment:
It seems like this line could be skipped.

Reply:
Thanks for the suggestion! I open a new PR to fix this.
https://github.com/JuliaSmoothOptimizers/NLPModelsModifiers.jl/pull/84